tough/src/schema/mod.rs

```diff
@@ -267,12 +301,28 @@ impl Target {
     }
 }
 
+impl Targets {
+    pub fn new(spec_version: String, version: NonZeroU64, expires: DateTime<Utc>) -> Self {
```

Comment:
These `new` functions probably address @iliana's concern in #20

Reply:
I don't currently implement `new()` for all of these types unfortunately. :(